content/source/docs/cloud/run/cli.html.md

```diff
@@ -52,17 +52,11 @@ terraform {
 }
 ```
 
-A Terraform Cloud [user API token](../users-teams-organizations/users.html#api-tokens) is also needed. It should be set as `credentials` in the [CLI config file](/docs/commands/cli-config.html#credentials). User tokens can be created in the [user settings](../users-teams-organizations/users.html#user-settings).
-
-```hcl
-credentials "app.terraform.io" {
-  token = "xxxxxx.atlasv1.zzzzzzzzzzzzz"
-}
-```
+Next, run `terraform login` to authenticate with Terraform Cloud. Alternatively, you can [manually configure credentials in the CLI config file](/docs/commands/cli-config.html#credentials).
 
 The backend can be initialized with `terraform init`.
 
-```shell
+```
```

Comment:
This code block (and similar below) is not actually shell after the first line, so the syntax highlighting was spurious and confusing.